### ISSUE TITLE
DF Manual Rolls: Initiative issue when toggled off in V9

### ISSUE THREAD

**Issue by ravingdragoon** [flamewave000/dragonflagon-fvtt#262]
**Module**
DF Manual Rolls 2.2.4

**Describe the issue**
Unable to roll initiative with the Manual Rolls toggle feature enabled but toggled off.

**To Reproduce**
Enable to Manual Rolls toggle
2. Set the toggle to "off"
3. Attempt to roll initiative

**Expected behavior**
Initiative should roll normally when Manual Rolls is toggled off.

**Screenshots (Optional)**
```foundry.js:6687
Uncaught (in promise) Error: The Roll has already been evaluated and is now immutable
[Detected 2 packages: df-manual-rolls, lib-wrapper]
    at Roll.evaluate (foundry.js:6687)
    at Combat.rollInitiative (foundry.js:15589)
    at Z.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:502)
    at Combat.<anonymous> (df-manual-rolls.js:5)
    at Generator.next (<anonymous>)
    at df-manual-rolls.js:5
    at new Promise (<anonymous>)
    at __awaiter$2 (df-manual-rolls.js:5)
    at Combat._Combat_rollInitiative (df-manual-rolls.js:5)
    at Combat.🎁Combat.prototype.rollInitiative#0 (libWrapper-wrapper.js:182)
    at CombatTracker._onCombatantControl (foundry.js:58305)
    at HTMLAnchorElement.<anonymous> (foundry.js:58198)
    at HTMLAnchorElement.dispatch (jquery.min.js:2)
    at HTMLAnchorElement.v.handle (jquery.min.js:2)
evaluate	@	foundry.js:6687
rollInitiative	@	foundry.js:15589
🎁call_wrapped	@	libWrapper-wrapper.js:502
(anonymous)	@	df-manual-rolls.js:5
(anonymous)	@	df-manual-rolls.js:5
__awaiter$2	@	df-manual-rolls.js:5
_Combat_rollInitiative	@	df-manual-rolls.js:5
🎁Combat.prototype.rollInitiative#0	@	libWrapper-wrapper.js:182
_onCombatantControl	@	foundry.js:58305
(anonymous)	@	foundry.js:58198
dispatch	@	jquery.min.js:2
v.handle	@	jquery.min.js:2
```

**Comment by plutoneld** [comment]
Can't reproduce in DF Manual Rolls v2.3 and FVTT v9.

**Comment by flamewave000** [comment]
Ah I missed this one when I was looking for reported issues relating to V9. I did fix this with issue #271
